src/java/org/apache/cassandra/gms/VersionedValue.java

```diff
@@ -79,6 +81,8 @@ public class VersionedValue implements Comparable<VersionedValue>
 
     // values for ApplicationState.REMOVAL_COORDINATOR
     public final static String REMOVAL_COORDINATOR = "REMOVER";
+    public final static List<String> BOOTSTRAPPING_STATES = Arrays.asList(VersionedValue.STATUS_BOOTSTRAPPING,
```

Comment:
nit: status to match the upstream patch?

Reply:
fixed